src/Admin/Views/Providers/Index.cshtml

```diff
@@ -15,9 +18,12 @@
             <button type="submit" class="btn btn-primary mb-2" title="Search"><i class="fa fa-search"></i> Search</button>
         </form>
     </div>
-    <div class="col-auto">
-        <a asp-action="Create" class="btn btn-secondary">Create Provider</a>
-    </div>
+    @if (AccessControlService.UserHasPermission(Permission.Provider_Create))
```

Comment:
suggestion (non-blocking): it seems like usually we've been declaring a variable in the code block at the top, can we make this consistent with the other pages?

Reply:
Thank you! I've updated this and the other instances to follow the convention of setting the variable in the code block.